logexporter/Makefile

```diff
@@ -14,7 +14,7 @@
 
 PROJECT = google-containers
```

Comment:
I would suggest changing this project to one here - https://github.com/kubernetes/perf-tests/blob/master/perfdash/Makefile#L6 - in order to avoid all extra hassle in pushing new image to prod.